NuGet.config

```diff
@@ -2,9 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-corefx-4ac4c03" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" />
-    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```

Comment:
@dougbu This seems to have been added directly to master by https://github.com/aspnet/Extensions/pull/2531 by @dotnet-maestro, so I'm going to leave it for now.